.README.md

```diff
@@ -22,6 +22,7 @@ This page registers books with author and title of the book. To register a book:
 2. Type the book's title.
 3. Click on Add button below.
 4. The books will be added to the list and will be displayed.
+5. added a navigation bar
 To remove a book from the list:
 1. Find the book in the list.
 2. Click on remove button for the respective book.
```

Comment:
- [OPTIONAL] To fix the issue with your readme file, please remove the dot before it's name, it should be `README.md` and not `.README.md`.